tests/baselines/reference/parserErrorRecovery_ObjectLiteral2.js

```diff
@@ -3,4 +3,5 @@ var v = { a
 return;
 
 //// [parserErrorRecovery_ObjectLiteral2.js]
-var v = { a: a, "return":  };
+var v = { a: a,
+    "return":  };

```

Comment:
These JS baseline changes are reverting back to their original state pre-https://github.com/microsoft/TypeScript/pull/36688. That PR was never really meant to change JS emit, but the couple changes that did occur seemed ok so we went with it. Since `siblingNodePositionsAreComparable` might do a little more work than what’s currently happening in master, I reverted the codepath that non-codefix/refactor emit takes back to the way it originally was.

Reply:
`siblingNodePositionsAreComparable` is now quite cheap, but I think it still makes sense to keep it out of JS emit for now.